tools/decompme

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python3
+
+import argparse
+import sys
+from glob import glob
+from pathlib import Path
+import json
+import requests
+
+# m2ctx.py is an optional dependency for context generation
+try:
+    import m2ctx
+except ImportError:
+    m2ctx = None
+
+DECOMP_ME_HOST = "https://decomp.me"
+SRC_DIR = "src/"
+NONMATCHINGS_DIR = "ver/us/asm/nonmatchings/"
+NONMATCHINGS_ASM_GLOB = NONMATCHINGS_DIR + "**/*.s"
+SYMBOL_ADDRS_TXT = "ver/us/symbol_addrs.txt"
+SCRATCH_DEFAULTS = {
+    "project": "papermario",
+    "platform": "n64",
+    "compiler": "gcc2.8.1",
+    "compiler_flags": "-O2 -fforce-addr",
+}
```

Comment:
This stuff should be a settings file (yaml?) that decomp.me can use when running its own version of the script, avoiding the need for us to run foreign code